README.md

```diff
@@ -934,6 +934,14 @@ The best places to start are the main entry points into Bolt:
 If you have additional notes that could be helpful for others, please submit
 them via pull request.
 
+## Known Issue
+
+- Linux kernel releases [ext4: fast commits](https://lwn.net/Articles/842385/)
+  feature in v5.10. It's a new, lighter-weight journaling method to reduce
+  unrelated IO for fsync/fdatasync. However, `fast commits` is new feature and
+  users run into data loss issue after power failure. The data loss cat corrupt
+  boltdb. If you enable fast commits feature on ext4 filesystem, please ensure
+  that the kernel includes related fix patches to fast commit. Details in [issue 562](https://github.com/etcd-io/bbolt/issues/562).
```

Comment:
- Please add an item in the content
- Please also provide a link to the fast commit issue, and also clarify which linux kernel versions contain the fix or a related link for users reference.

Reply:
Updated. Please take a look. Thanks